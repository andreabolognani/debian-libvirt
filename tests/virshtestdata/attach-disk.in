attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype block
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype nothing
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type disk
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype block --type disk
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type cdrom
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype block --type cdrom
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type blah
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype block --type blah
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type disk --driver testdriver
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type disk --subdriver qcow2
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type disk --subdriver raw
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type disk --cache none
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type disk --subdriver qcow2 --cache none
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type disk --subdriver qcow2 --serial TEST_SERIAL
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type disk --mode readonly
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type disk --mode shareable
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type disk --mode whatever
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type disk --subdriver qcow2 --alias testalias
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type disk --rawio
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type disk --multifunction
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type disk --subdriver qcow2 --alias testalias --iothread 3 --mode readonly --cache none --driver qemu

attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type disk --address ide:1.2.3
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type disk --address ide:1.2.4
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type disk --address ide:1:2:5
attach-disk --print-xml --domain testdom --source /nonexistent/file --target sda --sourcetype file --type disk --address ide:1.2.3
attach-disk --print-xml --domain testdom --source /nonexistent/file --target vda --sourcetype file --type disk --address ide:1.2.3

attach-disk --print-xml --domain testdom --source /nonexistent/file --target sda --sourcetype file --type disk --address usb:12.34
attach-disk --print-xml --domain testdom --source /nonexistent/file --target sda --sourcetype file --type disk --address usb:12.3
attach-disk --print-xml --domain testdom --source /nonexistent/file --target sda --sourcetype file --type disk --address usb:12:34
attach-disk --print-xml --domain testdom --source /nonexistent/file --target vda --sourcetype file --type disk --address usb:12.34
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type disk --address usb:12.34

attach-disk --print-xml --domain testdom --source /nonexistent/file --target sda --sourcetype file --type disk --address scsi:1.2.3
attach-disk --print-xml --domain testdom --source /nonexistent/file --target sda --sourcetype file --type disk --address scsi:1.2.4
attach-disk --print-xml --domain testdom --source /nonexistent/file --target sda --sourcetype file --type disk --address scsi:1:2:5
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type disk --address scsi:1.2.3
attach-disk --print-xml --domain testdom --source /nonexistent/file --target vda --sourcetype file --type disk --address scsi:1.2.3

attach-disk --print-xml --domain testdom --source /nonexistent/file --target sda --sourcetype file --type disk --address sata:1.2.3
attach-disk --print-xml --domain testdom --source /nonexistent/file --target sda --sourcetype file --type disk --address sata:1.2.4
attach-disk --print-xml --domain testdom --source /nonexistent/file --target sda --sourcetype file --type disk --address sata:1:2:5
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type disk --address sata:1.2.3
attach-disk --print-xml --domain testdom --source /nonexistent/file --target vda --sourcetype file --type disk --address sata:1.2.3

attach-disk --print-xml --domain testdom --source /nonexistent/file --target vda --sourcetype file --type disk --address pci:12.34.56.78
attach-disk --print-xml --domain testdom --source /nonexistent/file --target vda --sourcetype file --type disk --address pci:12:34:56:78
attach-disk --print-xml --domain testdom --source /nonexistent/file --target vda --sourcetype file --type disk --address pci:12.34.56.aa
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type disk --address pci:12.34.56.aa
attach-disk --print-xml --domain testdom --source /nonexistent/file --target sda --sourcetype file --type disk --address pci:12.34.56.aa

attach-disk --print-xml --domain testdom --source /nonexistent/file --target vda --sourcetype file --type disk --address pci:12.34.56.78 --multifunction
attach-disk --print-xml --domain testdom --source /nonexistent/file --target vda --sourcetype file --type disk --address pci:12:34:56:78 --multifunction
attach-disk --print-xml --domain testdom --source /nonexistent/file --target vda --sourcetype file --type disk --address pci:12.34.56.aa --multifunction

attach-disk --print-xml --domain testdom --source /nonexistent/file --target vda --sourcetype file --type disk --address ccw:12.34.56
attach-disk --print-xml --domain testdom --source /nonexistent/file --target vda --sourcetype file --type disk --address ccw:12:34:56
attach-disk --print-xml --domain testdom --source /nonexistent/file --target vda --sourcetype file --type disk --address ccw:12.34.56
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype file --type disk --address ccw:12.34.56
attach-disk --print-xml --domain testdom --source /nonexistent/file --target sda --sourcetype file --type disk --address ccw:12.34.56

attach-disk --print-xml --domain testdom --source /nonexistent/file --target vda --sourcetype file --type disk --address test:12.34.56
attach-disk --print-xml --domain testdom --source /nonexistent/file --target vda --sourcetype file --type disk --address test:12:34:56
attach-disk --print-xml --domain testdom --source /nonexistent/file --target vda --sourcetype file --type disk --address test:12.34.56

attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --source-protocol AAA --sourcetype file
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype network
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype network --source-protocol TEST --source-host-name hostname
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype network --source-protocol TEST --source-host-name hostname:port
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype network --source-protocol TEST --source-host-name hostname:
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype network --source-protocol TEST --source-host-name :port
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype network --source-protocol TEST --source-host-name :
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype network --source-protocol TEST --source-host-name hostname:port --source-host-transport trnsp
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype network --source-protocol TEST --source-host-transport trnsp
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype network --source-protocol TEST --source-host-transport trnsp --source-host-socket /nonexistent/socket
attach-disk --print-xml --domain testdom --source /nonexistent/file --target hda --sourcetype network --source-protocol TEST --source-host-socket /nonexistent/socket
