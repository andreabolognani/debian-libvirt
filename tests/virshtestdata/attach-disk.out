<disk type='file'>
  <source file='/nonexistent/file'/>
  <target dev='hda'/>
</disk>

<disk type='file'>
  <source file='/nonexistent/file'/>
  <target dev='hda'/>
</disk>

<disk type='block'>
  <source dev='/nonexistent/file'/>
  <target dev='hda'/>
</disk>

error: Unknown source type: 'nothing'

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='hda'/>
</disk>

<disk type='block' device='disk'>
  <source dev='/nonexistent/file'/>
  <target dev='hda'/>
</disk>

<disk type='file' device='cdrom'>
  <source file='/nonexistent/file'/>
  <target dev='hda'/>
</disk>

<disk type='block' device='cdrom'>
  <source dev='/nonexistent/file'/>
  <target dev='hda'/>
</disk>

<disk type='file' device='blah'>
  <source file='/nonexistent/file'/>
  <target dev='hda'/>
</disk>

<disk type='block' device='blah'>
  <source dev='/nonexistent/file'/>
  <target dev='hda'/>
</disk>

<disk type='file' device='disk'>
  <driver name='testdriver'/>
  <source file='/nonexistent/file'/>
  <target dev='hda'/>
</disk>

<disk type='file' device='disk'>
  <driver type='qcow2'/>
  <source file='/nonexistent/file'/>
  <target dev='hda'/>
</disk>

<disk type='file' device='disk'>
  <driver type='raw'/>
  <source file='/nonexistent/file'/>
  <target dev='hda'/>
</disk>

<disk type='file' device='disk'>
  <driver cache='none'/>
  <source file='/nonexistent/file'/>
  <target dev='hda'/>
</disk>

<disk type='file' device='disk'>
  <driver type='qcow2' cache='none'/>
  <source file='/nonexistent/file'/>
  <target dev='hda'/>
</disk>

<disk type='file' device='disk'>
  <driver type='qcow2'/>
  <source file='/nonexistent/file'/>
  <target dev='hda'/>
  <serial>TEST_SERIAL</serial>
</disk>

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='hda'/>
  <readonly/>
</disk>

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='hda'/>
  <shareable/>
</disk>

error: No support for whatever in command 'attach-disk'

<disk type='file' device='disk'>
  <driver type='qcow2'/>
  <source file='/nonexistent/file'/>
  <target dev='hda'/>
  <alias name='testalias'/>
</disk>

<disk type='file' device='disk' rawio='yes'>
  <source file='/nonexistent/file'/>
  <target dev='hda'/>
</disk>

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='hda'/>
</disk>

<disk type='file' device='disk'>
  <driver name='qemu' type='qcow2' iothread='3' cache='none'/>
  <source file='/nonexistent/file'/>
  <target dev='hda'/>
  <readonly/>
  <alias name='testalias'/>
</disk>

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='hda'/>
  <address type='drive' controller='1' bus='2' unit='3'/>
</disk>

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='hda'/>
  <address type='drive' controller='1' bus='2' unit='4'/>
</disk>

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='hda'/>
  <address type='drive' controller='1' bus='2' unit='5'/>
</disk>

error: expecting a scsi:00.00.00 or usb:00.00 or sata:00.00.00 address.

error: expecting a pci:0000.00.00.00 or ccw:00.0.0000 address.

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='sda'/>
  <address type='usb' bus='12' port='34'/>
</disk>

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='sda'/>
  <address type='usb' bus='12' port='3'/>
</disk>

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='sda'/>
  <address type='usb' bus='12' port='34'/>
</disk>

error: expecting a pci:0000.00.00.00 or ccw:00.0.0000 address.

error: expecting an ide:00.00.00 address.

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='sda'/>
  <address type='drive' controller='1' bus='2' unit='3'/>
</disk>

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='sda'/>
  <address type='drive' controller='1' bus='2' unit='4'/>
</disk>

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='sda'/>
  <address type='drive' controller='1' bus='2' unit='5'/>
</disk>

error: expecting an ide:00.00.00 address.

error: expecting a pci:0000.00.00.00 or ccw:00.0.0000 address.

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='sda'/>
  <address type='drive' controller='1' bus='2' unit='3'/>
</disk>

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='sda'/>
  <address type='drive' controller='1' bus='2' unit='4'/>
</disk>

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='sda'/>
  <address type='drive' controller='1' bus='2' unit='5'/>
</disk>

error: expecting an ide:00.00.00 address.

error: expecting a pci:0000.00.00.00 or ccw:00.0.0000 address.

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='vda'/>
  <address type='pci' domain='0x0012' bus='0x34' slot='0x56' function='0x78'/>
</disk>

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='vda'/>
  <address type='pci' domain='0x0012' bus='0x34' slot='0x56' function='0x78'/>
</disk>

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='vda'/>
  <address type='pci' domain='0x0012' bus='0x34' slot='0x56' function='0xaa'/>
</disk>

error: expecting an ide:00.00.00 address.

error: expecting a scsi:00.00.00 or usb:00.00 or sata:00.00.00 address.

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='vda'/>
  <address type='pci' domain='0x0012' bus='0x34' slot='0x56' function='0x78' multifunction='on'/>
</disk>

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='vda'/>
  <address type='pci' domain='0x0012' bus='0x34' slot='0x56' function='0x78' multifunction='on'/>
</disk>

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='vda'/>
  <address type='pci' domain='0x0012' bus='0x34' slot='0x56' function='0xaa' multifunction='on'/>
</disk>

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='vda'/>
  <address type='ccw' cssid='0x12' ssid='0x34' devno='0x0056'/>
</disk>

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='vda'/>
  <address type='ccw' cssid='0x12' ssid='0x34' devno='0x0056'/>
</disk>

<disk type='file' device='disk'>
  <source file='/nonexistent/file'/>
  <target dev='vda'/>
  <address type='ccw' cssid='0x12' ssid='0x34' devno='0x0056'/>
</disk>

error: expecting an ide:00.00.00 address.

error: expecting a scsi:00.00.00 or usb:00.00 or sata:00.00.00 address.

error: Invalid address.

error: Invalid address.

error: Invalid address.

error: --source-protocol option requires --sourcetype network

error: --source-protocol option requires --sourcetype network

<disk type='network'>
  <source protocol='TEST' name='/nonexistent/file'>
    <host name='hostname'/>
  </source>
  <target dev='hda'/>
</disk>

<disk type='network'>
  <source protocol='TEST' name='/nonexistent/file'>
    <host name='hostname' port='port'/>
  </source>
  <target dev='hda'/>
</disk>

<disk type='network'>
  <source protocol='TEST' name='/nonexistent/file'>
    <host name='hostname' port=''/>
  </source>
  <target dev='hda'/>
</disk>

<disk type='network'>
  <source protocol='TEST' name='/nonexistent/file'>
    <host name='' port='port'/>
  </source>
  <target dev='hda'/>
</disk>

<disk type='network'>
  <source protocol='TEST' name='/nonexistent/file'>
    <host name='' port=''/>
  </source>
  <target dev='hda'/>
</disk>

<disk type='network'>
  <source protocol='TEST' name='/nonexistent/file'>
    <host transport='trnsp' name='hostname' port='port'/>
  </source>
  <target dev='hda'/>
</disk>

<disk type='network'>
  <source protocol='TEST' name='/nonexistent/file'>
    <host transport='trnsp'/>
  </source>
  <target dev='hda'/>
</disk>

<disk type='network'>
  <source protocol='TEST' name='/nonexistent/file'>
    <host transport='trnsp' socket='/nonexistent/socket'/>
  </source>
  <target dev='hda'/>
</disk>

error: Option --source-host-transport is required by option --source-host-socket


## Exit code: 1
